src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts

```diff
@@ -217,24 +219,69 @@ if it is ++ or -- operater navigate to the previous or next tab
         },
       },
     },
-    {
-      id: 10,
-      label: 'Additional risks',
-      link: buildDynamicPath(
-        isEditMode
-          ? routes.PIA_ADDITIONAL_RISKS_EDIT
-          : routes.PIA_ADDITIONAL_RISKS_VIEW,
-        {
-          id: pia?.id,
-        },
-      ),
-      enable: showPostIntakeTabs,
-      state: {
-        prev: {
-          condition: true,
-          action: -1,
-        },
-      },
-    },
+    ...(!isMPORole()
```

Comment:
can you please memoize the value of isMPORole() with dependents to avoid continuous storage calls during re-renders?